docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,39 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint.
+
+**Subprojects** and **Cohorts** are terms used interchangeably in LORIS. **Subprojects** are defined in the `subproject` table of the database.
+
+
+## Adding Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page.
+  
+   You then click on _New SubprojectID_, fill in the fields on the right, click _save_. Refresh the page to view your new subproject. If you accidentally create an extra entry, you will have to delete it from the database manually with an sql command.  This process is similar to adding new projects.
+   
+> Note: Clicking _save_ more than once will register a duplicate subproject ID.
+
+### SQL
+Subprojects can be added directly in SQL using the following command. The `WindowDifference` and `RecruitmentTarget` are optional but should be set when the information is available.
```

Comment:
Should specify what the words mean in this context.

Reply:
I agree with @driusan.

Reply:
"Subprojects can be added directly in SQL using the following command. The `WindowDifference`, which refers to the study-defined candidate age range, and `RecruitmentTarget`, which refers to the study's target number of candidates to be recruited, are optional but should be set when the information becomes available."

Is this sufficient?

Reply:
@cmadjar @driusan As i mentioned in another doc. describing fields throughout the docs might make it impossible to maintain these documents in the longterm. I'd rather just remove the entire sentence.